rescript

```diff
@@ -417,56 +401,51 @@ if (
         fileName.endsWith(".js") ||
         fileName.endsWith(".mjs") ||
         fileName.endsWith(".cjs") ||
-        fileName.endsWith(".gen.tsx") ||
-        (genTypeFileExtension && fileName.endsWith(genTypeFileExtension)) ||
-        watch_generated.indexOf(fileName) >= 0 ||
+        fileName.endsWith(genTypeFileExtension) ||
+        watchGenerated.indexOf(fileName) >= 0 ||
         fileName.endsWith(".swp")
       );
     }
+
     /**
      * @return {boolean}
      */
     function needRebuild() {
-      return reasons_to_rebuild.length != 0;
+      return reasonsToRebuild.length !== 0;
     }
-    var error_is_tty = process.stderr.isTTY;
-    var std_is_tty = process.stdout.isTTY;
+
     function logFinish(code) {
+      let log = `>>>> Finish compiling`;
+      if (code !== 0) {
+        log = log + " (exit: " + code + ")";
+      }
       if (std_is_tty) {
-        if (code === 0) {
-          console.log(
-            "\x1b[36m>>>> Finish compiling\x1b[0m",
-            Math.floor(updateFinishTime() / 1e6),
-            "mseconds"
-          );
-        } else {
-          console.log(
-            "\x1b[1;31m>>>> Finish compiling(exit: " + code + ")\x1b[0m"
-          );
-        }
+        log = "\x1b[36m" + log + "\x1b[0m";
+      }
+      if (code !== 0) {
+        console.log(log);
```

Comment:
It might make sense to display build time even when it failed, but I've decided to keep it as it is.